### PR TITLE
Update country-info.csv

### PR DESCRIPTION
**Pull request by jffg**
Inria is the French national public institute for computer science www.inria.fr

# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

_Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

**Inclusion criteria**

- [ ] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

**Comment by emeryberger** [comment]
INRIA is not in the database. Are you planning to add it?

File: country-info.csv [hunk 1]
```diff
@@ -77,6 +77,7 @@ IIT Ropar,asia
 IMDEA Networks Institute,europe
 IMDEA Software Institute,europe
 IMSc,asia
+Inria, europe
```
Comment (emeryberger) [inline country-info.csv:80]: spurious space. Also, should be INRIA. Also, it's not in the database.

**Comment by jffg** [comment]
Dear colleague,

thank you for a swift answer to my pull request. 

Sorry for the spurious space before europe
About the writing: Inria is the modern way to write it, but we can use INRIA if you prefer.

Could you please tell me how I can add Inria to the database?

Several colleagues (as Serge Abiteboul, Francis Bach, etc.) are actually Inria researchers, but appear with a different affiliation in the ranking. 

Thanks again,
Jean-Frédéric 
--
Jean-Frédéric Gerbeau
Deputy CEO for Science, Inria
http://www.inria.fr <http://www.inria.fr/>

**Comment by emeryberger** [comment]
You would need a pull request updating `csrankings.csv` with all faculty members who can solely advise a PhD student _in computer science_ and who are full-time (>= 75%) affiliated with Inria (at a single physical location). I know Inria has a complicated setting that might make it difficult to include.

**Comment by jffg** [comment]
Thanks for your reply. 

Inria is a national public institute. Many of our researchers also have an affiliation with a university. They are nevertheless paid >= 75% by Inria (most often 100%).

I can ask my IT department to add all those who are not yet in your database, satisfying your criteria. 
But what about those who are already in the database with their university affiliation? 

Ex: 
Francis Bach,Ecole Normale Superieure,http://www.di.ens.fr/~fbach,6PJWcFEAAAAJ <http://www.di.ens.fr/~fbach,6PJWcFEAAAAJ>
is not paid >=75% by Ecole Normale Superieure, but by Inria. 

Shall we leave this line in the file and add a new one? Or shall we replace the line with a new one? 

Thanks,
--
Jean-Frédéric Gerbeau
Deputy CEO for Science, Inria
http://www.inria.fr

**Comment by emeryberger** [comment]
Please fix any incorrect affiliations.

**Comment by emeryberger** [comment]
I'm going to close this in anticipation of a later PR for all of INRIA. Thanks / Merci d'advance.

**Pull request closed without merging**